paths/electron/starting-with-electron/08-add-to-js.md

```diff
@@ -0,0 +1,30 @@
+---
+layout: simple-class
+header:
+  overlay_image: cover.jpeg
+  overlay_filter: rgba(46, 129, 200, 0.6)
+title: Add JavaScript
+permalink: /electron/starting-with-electron/add-to-javascript
+next-page: /electron/starting-with-electron/celebrate
+facilitator: false
+sidebar:
+  nav: "electron-getting-starting"
+main-content: |
+
+  Finally, let's add the fun part: the Javascript!
+
+  1. Copy the Javascript from Codepen into the `src/script.js` file.
+  1. Spin up your app, and see how it works.
+  1. Save and commit your changes.
```

Comment:
Okay, I figured this out! The problem was pretty complex, but the solution seems pretty straightforward. 

The reason that the App wasn't working was due to its use of JQuery, which [isn't as easily supported by default](https://electron.atom.io/docs/faq/#i-can-not-use-jqueryrequirejsmeteorangularjs-in-electron) as we'd hope in Electron. 

Here are the successful steps I took (after much troubleshooting):
- Downloaded the JQuery library.
- Added it to the repository.
- Followed [these instructions](https://stackoverflow.com/a/34881388)
- Loaded the App. 

The Codepen Nathan and I used also required the download of a dependency or two, so it likely doesn't help us out in addressing added complexity. Should we just add these instructions into the repo? Is there a shorter workaround? 

Returning to this in the morning after we have a chance to sync during team meeting.